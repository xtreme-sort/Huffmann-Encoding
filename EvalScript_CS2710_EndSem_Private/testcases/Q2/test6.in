DeriveHuffmanTree
12
4 00111
6 0000
H 100
T 0001
V 0010
Y 1010
b 110
g 010
n 011
p 111
u 1011
x 00110
Quit
