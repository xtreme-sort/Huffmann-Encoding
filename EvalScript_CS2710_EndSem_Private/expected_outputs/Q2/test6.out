((((6,T),(V,(x,4))),(g,n)),((H,(Y,u)),(b,p)));
